spack/package.py

```diff
@@ -24,18 +24,41 @@ class Oommf(Package):
     contributors to OOMMF are Mike Donahue, and Don Porter.
 
     Summary taken from OOMMF documentation https://math.nist.gov/oommf/
+
+    OOMMF home page: "https://math.nist.gov/oommf/"
+
+    OOMMF as a git repository: https://github.com/fangohr/oommf
+
+    Versions ending with "-vanilla" indicate that the sources are taken
+    directly from https://math.nist.gov/oommf/dist/ . All other versions are
+    from the https://github.com/fangohr/oommf (which includes the "-vanilla"
+    sources, and adds additional OOMMF extensions). See
+    https://github.com/fangohr/oommf for details.
     """
 
     homepage = "https://math.nist.gov/oommf/"
-    url = "https://math.nist.gov/oommf/dist/oommf20a2_20200608-hotfix.tar.gz"
+    url = "https://github.com/fangohr/oommf/archive/refs/tags/20a1_20180930_ext.tar.gz"
```

Comment:
What's the difference between this "stand-alone" `url` and the `url` inside the `versions` below?

Reply:
`url` seems to be version specific so I guess we can't have the same `url` for multiple versions?

Reply:
The stand-alone is the default URL. It will be overridden by the url in the `version` if one is given there.

We can use the same (default) URL for multiple versions: spack is clever enough to only use the path of the archive URL. (Actually, the default URL, versions and checksums are created automatically when you use `spack create NAME-OF-PACKAGE URL-OF-A-TARBALL`.